README.md

```diff
@@ -78,50 +79,49 @@ Update-MaesterTests
 
 ## Use as GitHub action
 
-Maester is also published to the [GitHub marketplace](https://github.com/marketplace/actions/maester-action) and can be used directly in any GitHub workflow.
+Maester is also published to the [GitHub marketplace](https://github.com/marketplace/actions/run-maester) and can be used directly in any GitHub workflow. Because it is specially built for GitHub it nicely integrates with all the features of GitHub Actions, like uploading artifacts and writing a summary to the workflow run.
```

Comment:
Nit pick: The editor in me suggests adding a comma after "built for GitHub[,]...". ✏️